pkg/controller/pxcbackup/controller.go

```diff
@@ -235,20 +229,8 @@ type VolumeStatus string
 
 const (
 	VolumeUndefined VolumeStatus = "Undefined"
```

Comment:
Looks there is no need in this const too. As well as the `VolumeStatus` type at all.